integration_tests/projects/002_jaffle_shop/models/schema.yml

```diff
@@ -87,3 +87,12 @@ models:
         description: Amount of the order (AUD) paid for by gift card
         tests:
           - not_null
+
+  - name: orders_daily
+
+  - name: orders_forecast
+    meta:
+      fal:
+        scripts:
+          after:
+            - models/orders_forecast.py
```

Comment:
its not really after right? We need to find a better mechanism to do this.. I think both approaches are valid, to create a new model and to update the existing one

Reply:
It does work as an `after` because making the model `ephemeral` makes it not actually run in dbt but mark it as run. So this works great